apps/solution/src/components/EditPerson/UsePerson.tsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react';
+import { getPerson, PersonModel, updatePerson } from '../../api/person';
+
+export default function usePerson(id: string) {
+  const [person, setPerson] = useState<PersonModel | null>(null);
+
+  useEffect(() => {
+    getPerson(id).then(setPerson);
+  }, [id]);
+
+  const refreshPerson = (): void => {
+    getPerson(id).then(setPerson);
+  };
```

Comment:
I don't think we need to keep it if we don't use it. What do you think @JonathanGeoffroy ? I'm also not sure of a usage of this sort of function 🤔

Reply:
It's used by ReactQuery if I recall correctly, they call it a mutation. 
Maybe we'll never make use of it in this school, but I thought it might be handy

Reply:
Generally speaking, I would always suggest to remove dead code ; if `refreshPerson`  is not called at all, we should drop it IMO.
Yes, react-query (and other libs) provide a `refetch` function, but I think we should see it as "I want to fetch new data that isn't changed by myself in this app, but might be changed anyway (another user for example)". 
I'm afraid implementing `refetch` would lead newbies to think about "I changed the user, now I have to refetch it manually", which might be a bad practice in some cases.